packages/react-core/src/components/Modal/ModalContent.tsx

```diff
@@ -73,29 +76,37 @@ export const ModalContent: React.FunctionComponent<ModalContentProps> = ({
   const modalBoxHeader = header ? (
     <div className={css(titleStyles.title)}>{header}</div>
   ) : (
-    <ModalBoxHeader hideTitle={hideTitle}> {title} </ModalBoxHeader>
+    title && <div className={css(styles.modalBoxTitle)}>{title}</div>
```

Comment:
I think it might be good to keep the ModalBoxHeader and just update it. This way, existing consumers won't be too broken